modules/linear_boltzmann_solvers/lbs_solver/lbs_structs.h

```diff
@@ -193,11 +193,11 @@ enum class PhiSTLOption
 
 class LBSGroupset;
 
-using SetSourceFunction = std::function<void(const LBSGroupset& groupset,
-                                             std::vector<double>& q,
-                                             const std::vector<double>& phi,
-                                             const std::vector<double>& densities,
-                                             const SourceFlags source_flags)>;
+typedef std::function<void(const LBSGroupset& groupset,
+                           std::vector<double>& q,
+                           const std::vector<double>& phi,
+                           const SourceFlags source_flags)>
+  SetSourceFunction;
```

Comment:
I noticed that was this changed from `using ` to `typedef`.

Reply:
Done